tests/test_bench.py

```diff
@@ -35,13 +44,11 @@
 from pytket.extensions.qiskit import tk_to_qiskit
 from qiskit import QuantumCircuit
 
-test_qasm_output_path = "./test_output/"
+TEST_QASM_OUTPUT_PATH = "./test_output/"
 
 
 def test_configure_begin():
-    Path(test_qasm_output_path).mkdir(parents=True, exist_ok=True)
-    utils.set_qasm_output_path(test_qasm_output_path)
-    assert utils.get_qasm_output_path() == test_qasm_output_path
+    Path(TEST_QASM_OUTPUT_PATH).mkdir(parents=True, exist_ok=True)
```

Comment:
is that test even necessary anymore? I would imagine the directory to be created in the tests below if it does not exist, but maybe I am wrong.

Reply:
Unfortunately that is not the case, therefore I added this to the newly created fixture.